damnit/gui/main_window.py

```diff
@@ -1030,6 +1052,55 @@ def show_adeqt(self):
             self.adeqt_window = AdeqtWindow(ns, parent=self)
         self.adeqt_window.show()
 
+    def _toggle_theme(self):
+        """Toggle between light and dark themes."""
+        new_theme = Theme.DARK if self.dark_mode_action.isChecked() else Theme.LIGHT
```

Comment:
The triggered signal includes a checked parameter if you want it: https://doc.qt.io/qt-5/qaction.html#triggered

```suggestion
    def _toggle_theme(self, checked):
        """Toggle between light and dark themes."""
        new_theme = Theme.DARK if checked else Theme.LIGHT
```